README.md

```diff
@@ -1 +1,1146 @@
+
+![alt text](images/claim_check.png)
+
+The ultimate goal of the claim check EIP in our scenario is to validate the student's ID. Check luggage used to send unwanted data which not uses invalidation process, to the student's detail database. After the validation of the student's ID, the original data will add to the request again.
+
+### Sample scenario
+The sample scenario used to demonstrate the above three EI Patterns. Student sends a request with content student ID, student name, Student's city and gender. Then ‘content filter’ filters the student name, student's city and gender from the request and added them to a students details database. The filtered request contains only the student ID. Then filtered request goes to the 'Student ID validator'. It validates the incoming request and sends the validated request. Then validated request goes to the 'content enricher'. Content enricher uses used two databases as student's results details and student's personal details.
+Using student's ID, it maps the details which belong to a particular student from the tables and added particular data to the request. While adding the enriching data, it processes the data using JSON to JSON transformation. The output request of the scenario contains student ID, student name, Student's city, gender and student's results details.
+
+
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE 
+
+### Optional Requirements
+- Ballerina IDE plugins ([IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina), [VSCode](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina), [Atom](https://atom.io/packages/language-ballerina))
+- [Docker](https://docs.docker.com/engine/installation/)
+- [Kubernetes](https://kubernetes.io/docs/setup/)
+
+## Implementation
+
+### Create the project structure
+
+Ballerina is a complete programming language that supports custom project structures. Use the following package structure for this guide.
+
+```
+ └── guide
+     ├── message_transformation
+     │   └── message_transformation.bal
+     │  
+     └── tests
+         └── message_transformation_test.bal
+
+```
+
+Create the above directories in your local machine and also create empty `.bal` files.
+
+Open the terminal and navigate to `# eip-message-transformation/guide` and run the Ballerina project initializing toolkit.
+
+```bash
+   $ ballerina init
+```
+### Developing the SQL data
+
+Ballerina language has built-in support for writing web services. The `service` keyword in Ballerina simply defines a web service. Inside the service block, we can have all the required resources. You can define a resource inside the service. You can implement the business logic inside a resource using Ballerina language syntax.
+We can use the schema of the following database to store the student's data and student's results data.
+
+tsetdb database used to store student's data in student table as below.
```

Comment:
Shall we change this to 'The `tsetdb` database is used to store the student's data in the student table as shown below."

Can we reword all similar sentences in a similar way?

Reply:
Done